app/controllers/filters_controller.rb

```diff
@@ -12,7 +12,11 @@ def index
   end
 
   def new
-    @filter = resource_base.build
+    render 'react/index'
+  end
+
+  def edit
+    render 'react/index'
```

Comment:
I think there is a better way, and the rails page still gets loaded now in the buttom on the page, under the submit 
![Screenshot from 2022-04-12 18-33-49](https://user-images.githubusercontent.com/30431079/163010947-410c794d-0489-4191-af13-786a1dd0a1de.png)